playbook/liqo_incoming_peering.yaml

```diff
@@ -6,14 +6,14 @@
 # -----------------------------
 
 - name: Transfer kubeconfig from remote to local and prepare it
-  hosts: localhost
+  hosts: 127.0.0.1
```

Comment:
Hosts values are better not to be explicited into playbooks.
Consider defining localhost into Ansible inventory, or relying on the implicit `localhost` in Ansible.
More information, [here](https://docs.ansible.com/ansible/latest/inventory/implicit_localhost.html).